src/components/SourceSearch.js

```diff
@@ -58,13 +58,17 @@ const Search = React.createClass({
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
     shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.search.key")}`,
-      this.toggle);
+     this.toggle);
+    shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.searchAlt.key")}`,
+     this.toggle);
```

Comment:
ahh good idea! I was thinking of how to refactor. I will also update the style for the 80 chars limit.